docs/userguides/platform.md

```diff
@@ -20,6 +20,23 @@ The Platform UI will let you create and manage Clusters using a graphical experi
 The CLI experience is for those working locally who don't want to visit the website, or are locally developing their applications.
 ```
 
+Once you have created your Cluster, you have to fund it so it is made available for your use.
```

Comment:
question: Maybe I misunderstood how things were supposed to work. Doesn't funding the stream create the clsuter? Similar to chaosnet?

Reply:
It does but I need the record to exist in the database so I can use the cluster_id to connect the apepay stream back to the database in a more encrypted fashion

Plus, there may be more complicated arrangements (e.g. ability to create time-limited, "free clusters" for trial use) and also (as you mentioned) alternative payment systems that could come into play

Reply:
Maybe fine for now. But might be worth reworking this process a bit to make more traditional sense. Or rather, describe it better to the user. (e.g. create cluster, then make payment)

Reply:
Hmm, what do you think about refering to it like "reserve your cluster" or something that depicts the 2 stage process a bit better?